elixir/apps/web/test/support/conn_case.ex

```diff
@@ -233,4 +233,37 @@ defmodule Web.ConnCase do
     |> elements_to_text()
     |> Enum.reject(&(&1 in ["", "Previous", "Next", "Clear filters"]))
   end
+
+  ## Wait helpers
+
+  @doc """
+  Waits for an ExUnit assertion to be `true` before timing out.
+
+  This is helpful when we check UI state changes in LiveView tests,
+  where we don't know if the view was updated before or after the test.
+
+  Default wait time is 2 seconds.
+  """
+  def wait_for(assertion_callback, wait_seconds \\ nil, started_at \\ nil) do
```

Comment:
This isn't blocking, just a style question.  Just wondering why the default value for `wait_seconds` isn't defined here, but rather in the rescue block?  Is that just a personal preference or is there an advantage to doing it this way?

Reply:
@bmanifold you are right, I just copied this code from acceptance tests, pushed a fix for it in both places.

Reply:
👍